consensus/src/dag/dag_store.rs

```diff
@@ -323,7 +331,7 @@ impl Dag {
 
         let bitmask = self
             .nodes_by_round
-            .range(lowest_round..=target_round)
+            .range(lowest_round..target_round)
```

Comment:
what did this break?

Reply:
I added a verify function for the length and it rejects everything now 🤦